### PR TITLE
manifest: update Matter revision to pull bt_fixed_passkey fix.

### PR DESCRIPTION
**Pull request by ArekBalysNordic**
BT_FIXED_PASSKEY config was moved from prj_release.conf file to sdk-connectedhomeip space to avoid a warning during the release version compilation of the Matter Lock sample.

Also repaired another warning regarding PRINTK.

### CODE REVIEW

File: samples/matter/lock/prj_release.conf [hunk 1]
```diff
@@ -42,3 +41,4 @@ CONFIG_ASSERT_NO_FILE_INFO=y
 CONFIG_PRINTK=n
 CONFIG_PRINTK_SYNC=n
 CONFIG_THREAD_NAME=n
+CONFIG_BOOT_BANNER=n
```
[the remark this answers is not in the document]
Comment (ArekBalysNordic) [reply samples/matter/lock/prj_release.conf:44]: Done.

**Review by kkasperczyk-no — APPROVED** [review]

**Review by markaj-nordic — APPROVED** [review]
Isn't weather_station affected as well?

**Comment by kkasperczyk-no** [comment]
> Isn't weather_station affected as well?

I didn't check, but maybe not because we do not disable PRINTK in prj_release.conf. Apart from that I don't why we don't do disable it only for weather station.

**Review by LuDuda — APPROVED** [review]

**Comment by ArekBalysNordic** [comment]
> Isn't weather_station affected as well?

I've not seen that on the weather_station release build.

**Review by tejlmand — APPROVED** [review]

**Pull request merged**